### ISSUE TITLE
Support font size scaling for a11y

### ISSUE THREAD

**Issue by ManuelCalavera97** [material-components/material-web#1751]
**Describe the bug**

While using outlined attribute on the textfield component, the label gets missalinged

**Steps to reproduce the behavior:**

Deploy Open wc es dev server
Import textfield
Set label and outlined properties
Voula

**Browser Version:**

Chrome 85

<img width="558" alt="textfield" src="https://user-images.githubusercontent.com/45204611/91996174-fd30b280-ed38-11ea-9dd4-8528ed5bc717.PNG">

**Comment by asyncLiz** [comment]
I'm unable to reproduce this on Chrome 85. Can you provide a reproduction with jsbin or unpkg?

https://jsbin.com/tikanajaci/edit?html,output

**Comment by ManuelClavera97** [comment]
Alright, I guess I got it. The default css for open wc initialized application includes `:host {
 font-size: calc(10px + 2vmin);
}` which messes up with font size of the component.

So the question would be, should the component override all the existing font styles or is it problem on my part?

**Comment by asyncLiz** [comment]
Got it! I'll update the issue title to more accurately reflect the problem, which is font scaling. We're aware of this problem and it's something our designers have been tracking for a while now. Here are a couple of upstream issues you can follow for some more context:

https://github.com/material-components/material-components-web/issues/1932
https://github.com/material-components/material-components-web/issues/4164

The tricky part with font scaling is that it ties very closely with the density subsystem of Material Design. If the font gets smaller, the component usually gets smaller (negative density), and if the font gets bigger the component should get bigger (positive density).

The guidance we need from design is how we can support font scaling for a11y while maintaining strict guidelines around the density system, which is why we haven't implemented it just yet.

**Comment by asyncLiz** [comment]
All fonts in M3 now hook up to tokens, so you can explicitly set font type tokens and provide `rem` measurements instead of `px`
